rs3_trading/historicaldata/historical_process.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+import json
+import logging
+import urllib
+from collections import deque
+from time import sleep
+from typing import Any, Coroutine, Iterable
+
+import pandas as pd
+import requests
+
+from rs3_trading.database.dbcm import DuckDBCM
+from rs3_trading.historicaldata.historical_utils import extract_item_name_from_table, historical_data_transformation
+from rs3_trading.historicaldata.request_errors import check_request_status, NotFoundError, RequestTimeoutError, TooManyRequestsError
+from rs3_trading.utils.utilities import create_database, insert_into_table, join_with_commas
+
+DATABASE_NAME = 'GE_Tick_Data.db'
```

Comment:
The use of constants here is a bit concerning. I would start by making some of them parameters that are provided by your driving method (i.e. historical main). A good first one is database name, make historical main pass that in to the method